libraries/botbuilder-dialogs-adaptive-teams/src/schemas/actions/Teams.GetMeetingParticipant.schema

```diff
@@ -0,0 +1,54 @@
+{
```

Comment:
These .schema files are all just copy/paste from the Dotnet repo

Reply:
Sounds good to me

Reply:
Please use the ones from my branch, not the ones from 'main': https://github.com/microsoft/botbuilder-dotnet/pull/4782

Reply:
I think the `schemas` folder should be under the root folder of `botbuilder-dialogs-adaptive-teams` to align with other packages.

Reply:
@EricDahlvang I'll get them switched over.

@chon219 Yeah, I debated about where to put those since Dotnet also doesn't have a `src` folder. @stevengum Thoughts on whether the `schemas\` folder should be under the root or under `src\`?

Reply:
Ah yes, let's place the schemas outside of `src` to align with the other packages, e.g. [`botbuilder-ai@4.11.0`](https://github.com/microsoft/botbuilder-js/tree/4.11/libraries/botbuilder-ai). They're also just static files.

Good catch @chon219!

Reply:
understood. moved schemas